apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -30,11 +36,112 @@ import { getSiteConfig, getSiteNameAndCodeBuildId } from "../site/site.service"
 import { incrementVersion } from "../version/version.service"
 import { createDefaultPage } from "./page.service"
 
+const ajv = new Ajv({ allErrors: true, strict: false, logger: false })
```

Comment:
issue: suspect there might be some rebasing issue here. This `ajv` should be conflicting with the `ajv` import above at line 20.

Reply:
yea i fixe d it, was planning to check psot rebase :sadge: